src/modules/include/moduleManager.hpp

```diff
@@ -33,9 +33,9 @@ class ModuleManager {
             .Start = [&module]() { module.Start(); },
             .Setup = [&module](const configuration::ConfigurationParser& configurationParser) { module.Setup(configurationParser); },
             .Stop = [&module]() { module.Stop(); },
-            .ExecuteCommand = [&module](std::string query) -> Co_CommandExecutionResult
+            .ExecuteCommand = [&module](std::string query, nlohmann::json parameters) -> Co_CommandExecutionResult
```

Comment:
I think it will be clearer to name the first parameter `command`. Change this in all modules.

Reply:
I also noted that you are using `[[maybe_unused]]` in function declarations. They only make sense in function definitions. If possible I would take them out, though it doesn't make a big difference.

Reply:
Done!